swap_meet/vendor.py

```diff
@@ -1,2 +1,119 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = [] if inventory is None else inventory
+
+    def add(self, item):
+        self.inventory.append(item) 
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
```

Comment:
👍 Nice approach to look for the case that would prevent us from removing an item and handling that before getting to our main logic. This guards the main logic from a data configuration that would cause a problem. It also allows the main logic to be written without being indented, emphasizing its importance.